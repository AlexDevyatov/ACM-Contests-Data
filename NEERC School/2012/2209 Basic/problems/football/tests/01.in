3
2:1
0:9
1:1
