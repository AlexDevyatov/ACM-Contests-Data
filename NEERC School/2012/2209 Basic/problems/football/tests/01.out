62 37
0 99
53 46
